src/linear_algebra/finite_dimensional.lean

```diff
@@ -828,6 +828,18 @@ by { unfold findim, simp [dim_top] }
 
 end top
 
+lemma is_basis_of_findim_eq_zero [finite_dimensional K V]
+  (hV : findim K V = 0) : is_basis K (λ x : fin 0, (0 : V)) :=
+begin
+  split,
+  { rw linear_independent_iff',
+    intros,
+    exact fin.elim0 i },
+  { rw ← findim_top at hV,
+    rw [eq_top_iff, (@findim_eq_zero K V _ _ _ _ _).1 hV],
+    exact bot_le }
+end
```

Comment:
`is_basis_empty` is similar to this - it would probably make sense to copy its API by taking ` {ι : Type*} (h : ¬nonempty ι)` as an argument rather than using `fin 0`.

Reply:
I think I misunderstood, I thought you meant to prove the lemma using `is_basis_empty` which I have done with ```lemma is_basis_of_findim_eq_zero [finite_dimensional K V]
  (hV : findim K V = 0) : is_basis K (λ x : fin 0, (0 : V)) :=
is_basis_empty (finset.univ_eq_empty.mp rfl) (eq_zero_of_findim_eq_zero hV)``` but I'm not sure what you mean by taking {`ι : Type*} (h : ¬nonempty ι)` as an argument rather than using `fin 0`

Reply:
I mean change the statement to something like:
```lean
lemma is_basis_of_findim_eq_zero [finite_dimensional K V] {ι : Type*} (h : ¬nonempty ι)
  (hV : findim K V = 0) : is_basis K (λ x : ι, (0 : V)) :=
is_basis_empty h) (eq_zero_of_findim_eq_zero hV)
```